Add vector search param (#1353)
meilisearch/meilisearch-js-plugins: .changeset/strong-dodos-carry.md

---
"@meilisearch/instant-meilisearch": minor
---

> Add vector search param to support vector search with user-provided embedding

Reviewer: ```suggestion
Add `vector` field to the search params object to support user-provided embeddings

```
